### PR TITLE
Add react-ts Vite template

### PR DESCRIPTION
**Pull request by flanakin**
## 🛠️ Description
Adds the Vite react-ts template into `src/web`.

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [ ] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [ ] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [ ] ✅ Updated changelog (required for `dev` PRs)
> - [ ] ➡️ Will add log in a future PR (feature branch PRs only)
> - [x] ❎ Log not needed (small/internal change)

### 📖 Did you update documentation?

> - [ ] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [ ] ➡️ Will add docs in a future PR (feature branch PRs only)
> - [x] ❎ Docs not needed (small/internal change)

### CODE REVIEW

File: src/web/.gitignore
Comment (flanakin) [inline src/web/.gitignore:1]: @Ivanmtta Should we merge this with the root .gitignore file? Most of these are surprisingly not covered in that file.

File: src/web/package.json
Comment (flanakin) [inline src/web/package.json:1]: @Ivanmtta This adds a second package.json nested under the one at the root. For simplicity, I'm thinking we should probably only have one for now. What do you think? I can move these dependencies up or we can leave the nested packages if you feel that's best. 

An alternative might be to have a structure like this:
<root>
- tools
  - package.json
- web
  - package.json

Do you have a preference?

File: src/web/src/App.tsx
Comment (flanakin) [inline src/web/src/App.tsx:1]: @Ivanmtta I don't love having `src/web/src`. Would it make sense to move files up one level? That's the main reason why I had another comment about moving web to the root and renaming the current `src` to `tools`. Or should I just accept the two src folders?

**Review by Ivanmtta — APPROVED** [review]

**Comment by Ivanmtta** [comment]
Looks good! :)

**Pull request merged**